Source/NSBitmapImageRep+BMP.m

```diff
@@ -0,0 +1,444 @@
+/*
+   NSBitmapImageRep+PNG.m
+
+   Methods for loading .png images.
+
+   Copyright (C) 2003-2013 Free Software Foundation, Inc.
+   
+   Written by: Alexander Malmberg <alexander@malmberg.org>
```

Comment:
Sorry, everything in this file is just wrong. Could you please remove it.

Reply:
No, because it's an old file and it's been there for years.

Reply:
File removed.